snaps/reference/snaps-api.md

```diff
@@ -864,7 +864,9 @@ await snap.request({
 ## `snap_notify`
 
 Displays a [notification](../features/notifications.md) in MetaMask or natively in the OS.
-Snaps can trigger a short notification text for actionable or time sensitive information.
+Snaps can trigger a short notification text for actionable or time sensitive information. 
+`inApp` notifications can also include an optional title, content, and footer shown in an 
+expanded view when a user expands the notification.
 
 #### Parameters
```

Comment:
This can be split up into two sections: Parameters (without expanded view) and Parameters (with expanded view). Then it will be easier to document, you can have `type` and `message` required for without expanded view. Then with expanded view `type`, `message`, `title`, `content` are all required with `footerLink` being optional. You can then change "Example" to "Examples" and include a few code snippets including the different cases. 

On a side note, I do think our docs in general can benefit from more examples.